syntax = "proto3";

package parsed_hrb.v1;

message ceo_relation {
  string id = 1; 
  string company_id = 2; 
  string ceo_id = 3; 
}
